packages/server/graphql/mutations/addTeamMemberIntegrationAuth.ts

```diff
@@ -102,6 +103,16 @@ const addTeamMemberIntegrationAuth = {
         const manager = new GitLabOAuth2Manager(clientId, clientSecret, serverBaseUrl)
         tokenMetadata = await manager.authorize(oauthCodeOrPat, redirectUri)
       }
+      if (service === 'azureDevOps') {
+        if (!oauthVerifier) {
+          return {
+            error: {message: 'Missing OAuth2 Verifier required for Azure DevOps authentication'}
+          }
+        }
+        tokenMetadata = (await AzureDevOpsServerManager.init(oauthCodeOrPat, oauthVerifier)) as
```

Comment:
-1 Checking for `refreshToken` and asserting it exists should be done inside `init`

Reply:
This change has been made